### PR TITLE
Update tests and description

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
I've shamefully copied content from https://github.com/equinor/semeio/pull/77 and added it to the description of the schema. Furthermore the schema should then either be added using the generate rst function, or using the sphinx extension from config suite.

The tests have been slightly restructured as well, one test removed (`invalid_too_short_index_list`, as it did not test what was expected, outdated), and added one more for required keys.

Currently one test is failing because there are multiple errors when adding a negative value in the index list. The second error, which shouldn't be there, is:
`"'Will go through the input and try to convert to list of int' failed on input '[-1, 2, 3]' with error 'invalid literal for int() with base 10: '''"`

Not clear how to handle the last error yet.

### CODE REVIEW

**Comment by lars-petter-hauge** [comment]
I can split out the
```
if elem.startswith("-"):
    raise ValueError("Do no accept negative elements")
```
in a single commit..

File: semeio/jobs/correlated_observations_scaling/job_config.py [hunk 1]
```diff
@@ -44,10 +44,10 @@ def _realize_list(input_string):
     """
     real_list = []
     for elem in input_string.split(","):
+        if elem.startswith("-"):
+            raise ValueError("Do no accept negative elements")
```
Comment (oyvindeide) [inline semeio/jobs/correlated_observations_scaling/job_config.py:48]: Might want to reword this message a bit.

File: semeio/jobs/correlated_observations_scaling/job_config.py [hunk 2]
```diff
@@ -93,67 +169,36 @@ def build_schema():
...
-                            }
-                        },
-                    },
+                    "keys": _KEYS_SCHEMA,
                     "threshold": {
                         MK.Required: False,
                         MK.Type: types.Number,
                         MK.ElementValidators: (_min_max_value,),
+                        MK.Description: "Instructs the job to ignore principal"
+                        " components where the summative variance is smaller"
```
Comment (oyvindeide) [inline semeio/jobs/correlated_observations_scaling/job_config.py:180]: Perhaps use a different word than summative? Not very commonly used.

File: semeio/jobs/correlated_observations_scaling/job_config.py [hunk 3]
```diff
@@ -93,67 +169,36 @@ def build_schema():
...
+                        " than this value.",
+                    },
+                    "std_cutoff": {
+                        MK.Required: False,
+                        MK.Type: types.Number,
+                        MK.Description: "Filters out any observation that has"
```
Comment (oyvindeide) [inline semeio/jobs/correlated_observations_scaling/job_config.py:186]: Is this accurate enough? Is filtered out if the ensemble std is outside that value.

File: semeio/jobs/correlated_observations_scaling/job_config.py (hunk 1 above)
Comment (lars-petter-hauge) [reply semeio/jobs/correlated_observations_scaling/job_config.py:48]: Rephrased. Also note that there will be two errors thrown, and this is only from the transformation. The error from validation will still be output.

File: semeio/jobs/correlated_observations_scaling/job_config.py (hunk 2 above)
Comment (lars-petter-hauge) [reply semeio/jobs/correlated_observations_scaling/job_config.py:180]: changed with `cummulative`

File: semeio/jobs/correlated_observations_scaling/job_config.py (hunk 3 above)
Comment (lars-petter-hauge) [reply semeio/jobs/correlated_observations_scaling/job_config.py:186]: I think we should use the same descriptions for both this and the misfit preprocessor job. I'll copy those over here.

**Comment by lars-petter-hauge** [comment]
If you're happy with this I'll rebase :)

**Pull request merged**